nodejs/no-package-manager/server.js

```diff
@@ -2,16 +2,38 @@ const http = require('http')
 const port = process.env.PORT || 8080
 
 const requestHandler = (request, response) => {
+
+  const requestTime = new Date(Date.now()).toString();
+  console.log(request.method, request.headers.host, request.url, requestTime);
+
+  if (request.url === "/") {
+    response.writeHead(200, { "Content-Type": "text/html" });
     response.end(`<!DOCTYPE html>
 <html>
-  <head>
-    <title>Powered By Paketo Buildpacks</title>
-  </head>
-  <body>
-    <img style="display: block; margin-left: auto; margin-right: auto; width: 50%;" src="https://paketo.io/images/paketo-logo-full-color.png"></img>
-  </body>
-</html>`)
-}
+<head>
+  <title>Powered By Paketo Buildpacks</title>
+</head>
+<body>
+  <img style="display: block; margin-left: auto; margin-right: auto; width: 50%;" src="https://paketo.io/images/paketo-logo-full-color.png"></img>
+</body>
+</html>`);
+  } else if (
+    request.url === "/actuator/health/readiness" &&
```

Comment:
Just wondering how you chose those endpoints. I was looking at https://nodeshift.dev/nodejs-reference-architecture/operations/healthchecks/#endpoints and from that we might use /readyz and /livez

Reply:
I guess maybe it is to match the Java examples?

Reply:
Yes, to match the routes from the java example